common/src/Microsoft.Azure.IIoT.Validators.JsonSchemaDotNet/src/Default/JsonSchemaDotNetSchemaValidator.cs

```diff
@@ -0,0 +1,71 @@
+﻿namespace Microsoft.Azure.IIoT.Validators {
+    using Json.Pointer;
+    using Json.Schema;
+    using System.Collections.Generic;
+    using System.IO;   
+    using System.Text;
+    using System.Text.Json;
+    
+    /// <summary>
+    /// JsonSchema.Net schema validator.
+    /// </summary>
+    public class JsonSchemaDotNetSchemaValidator : IJsonSchemaValidator {
+
+        /// <inheritdoc/>
+        public IList<JsonSchemaValidationResult> Validate(byte[] jsonBuffer, TextReader schemaReader) {
```

Comment:
@suneetnangia should we guard with null checks on `jsonBuffer` and `schemaReader`? If we do intend for this to potentially be reused ... might be a nice add.

Reply:
Agreed, added as per CA1062. + I need to check why editorconfig is not playing well with VS, it should have thrown warning.